### PR TITLE
#371 explore countries, fix truncate reference when larger than 2 lines

### PR DESCRIPTION
**Pull request by dedandy**
truncate explore countries references word wihen exceed 2 lines and substitute with 3 dots when there are.

Only in dataset_list.html, not in dataset_details.html, where adjust only semantic html

Looks like this:

### CODE REVIEW

File: frontend/css/rodi_css.css [hunk 1]
```diff
@@ -359,6 +359,24 @@ h1 + .lead {
...
+
+/*
+==============================================================================
+    truncate lines
+==============================================================================
+*/
+
+
+
+.trunkate {
```
Comment (oncletom) [inline frontend/css/rodi_css.css]: There is already `.single-line` CSS class which does exactly this. And which works also for Firefox.

This snippet works only for Chrome.

**Comment by pzwsk** [comment]
Thanks for the tips Thomas,

also, requirement is to have similar height between rows whenever we have
or two lines.

Thanks

File: frontend/css/rodi_css.css (hunk 1 above)
Comment (pzwsk) [reply frontend/css/rodi_css.css]: True, yes

**Comment by oncletom** [comment]
So it's not a problem if that works only in Chrome?

This is the result in Firefox.

image

The output is likely to be similar in Edge and Safari.

**Comment by dedandy** [comment]
Maybe I have an idea to fix this problem... 😉 tomorrow I'll try and test chrome/ff

**Pull request merged**